backend/auth/Auth.ts

```diff
@@ -6,10 +6,28 @@ This is where we will authenticate users.
 */
 
 import { router } from "../Router.js";
-import passport from "passport";
-import { Strategy } from "passport-custom";
 import cookieSession from "cookie-session";
+import passport from "passport";
+import { Strategy } from "passport-local";
 import { renderWebApp } from "../WebApp/RenderWebApp";
+import { findOrCreateLocalUser, findUser } from "../users/Users";
+
+const LOCAL_DEV_USER_EMAIL = "localdev@email.com";
+
+let passportStrategy = new Strategy(async function (email, password, done) {
+  try {
+    let user = await findUser(email, password);
+    if (user) {
+      return done(null, user);
+    } else {
+      return done(null, false);
```

Comment:
Hmm I would have expected returning an error here - why are we calling the errback with `null` as the error when we weren't able to login?

```suggestion
      return done(Error("No such user"));
```

Reply:
I might be confused.  I thought, in Passport, failing to authenticate returns false instead of an error.   ?

Reply:
You are right - I was mistaken and we should call it with false when authentication fails.